eng/BootStrapMsBuild.targets

```diff
@@ -202,56 +202,32 @@
            AlwaysCreate="true" />
   </Target>
 
+  <!-- The task allow to download sdk bits for the specified version. It will be used later to bootstrap the runnable MSBuild. -->
+  <UsingTask TaskName="MSBuild.Bootstrap.Utils.Tasks.InstallDotNetCoreTask" AssemblyFile="$(ArtifactsBinDir)MSBuild.Bootstrap.Utils\$(Configuration)\netstandard2.0\MSBuild.Bootstrap.Utils.dll" TaskFactory="TaskHostFactory" />
   <Target Name="BootstrapNetCore" DependsOnTargets="CleanBootstrapFolder">
 
-    <!-- Publish the project first.  The more obvious way to do this would be to depend on the Publish target,
-         but that causes a target dependency cycle, even though it seems like AfterTargets="AfterBuild" should
-         probably not count as a link in the cycle. -->
-    <MSBuild Projects="$(MSBuildProjectFile)" Targets="Publish" BuildInParallel="$(BuildInParallel)" />
+    <PropertyGroup>
+      <InstallDir>$(ArtifactsBinDir)bootstrap\$(TargetFramework)\</InstallDir>
```

Comment:
We need it to differenciate between net472 and netcore, otherwise :) :
<img width="530" alt="image" src="https://github.com/dotnet/msbuild/assets/95473390/7b8f2cbf-14df-43a6-969d-62b41614d230">

Reply:
Can it be "core" or something that won't ever change then?